src/mutation_ui.cpp

```diff
@@ -23,28 +23,38 @@ void draw_exam_window( WINDOW *win, const int border_y )
     mvwputch( win, border_y, width - 1, BORDER_COLOR, LINE_XOXX );
 }
 
-void show_mutations_titlebar( WINDOW *window, std::string &menu_mode )
+const auto shortcut = []( const std::string &s )
 {
-    werase( window );
-
-    std::string caption = _( "MUTATIONS -" );
-    int cap_offset = utf8_width( caption ) + 1;
-    mvwprintz( window, 0,  0, c_blue, "%s", caption.c_str() );
+    return string_format( "<color_yellow>%s</color>", s.c_str() );
+};
 
-    std::string desc;
-    int desc_length = getmaxx( window ) - cap_offset;
+void show_mutations_titlebar( WINDOW *window, std::string &menu_mode, input_context &ctxt )
+{
+    werase( window );
 
+    std::ostringstream desc;
+    const int width = getmaxx( window );
     if( menu_mode == "reassigning" ) {
-        desc = _( "Reassigning.\nSelect a mutation to reassign or press SPACE to cancel." );
-    } else if( menu_mode == "activating" ) {
-        desc = _( "<color_green>Activating</color>  <color_yellow>!</color> to examine, <color_yellow>=</color> to reassign." );
-    } else if( menu_mode == "examining" ) {
-        desc = _( "<color_ltblue>Examining</color>  <color_yellow>!</color> to activate, <color_yellow>=</color> to reassign." );
+        desc << _( "Reassigning." ) << "  " <<
+             _( "Select a mutation to reassign or press <color_yellow>SPACE</color> to cancel. " );
+    }
+    if( menu_mode == "activating" ) {
+        desc << "<color_green>" << _( "Activating" ) << "</color>  " <<
```

Comment:
cout-style output makes localization more difficult. Here, the code assumes grammar order, which varies based on locale. Localizers would have to workaround this by finding the fragments i.e. there is coupling between the strings to localize, but the code explicitly hides that from the localizers. Consider using printf-style strings instead.